src/clj/rems/api/workflows.clj

```diff
@@ -52,10 +50,17 @@
       :return CreateWorkflowResponse
       (ok (workflow/create-workflow! (assoc command :user-id (getx-user-id)))))
 
+    (PUT "/edit" []
+      :summary "Edit workflow"
+      :roles #{:owner}
+      :body [command EditWorkflowCommand]
+      :return SuccessResponse
+      (ok (workflow/edit-workflow! command)))
+
     (PUT "/update" []
       :summary "Update workflow"
```

Comment:
These api docs need to be fixed. It was fine to have one endpoint that is described as "update workflow", but now that there is update and edit the difference needs to be explained. Consider something like "edit workflow title and settings" and "update workflow state (enabled, archived)"

Reply:
Done, the summaries are now more detailed.